docs/.eslintrc.cjs

```diff
@@ -4,73 +4,96 @@
  */
 
 module.exports = {
-	extends: [require.resolve("@fluidframework/eslint-config-fluid"), "prettier"],
 	parserOptions: {
-		project: ["./tsconfig.json"],
+		ecmaVersion: "latest",
```

Comment:
Required in order for the in-docs code-block linting to parse sample code correctly.

Reply:
Do we want to specify "latest"? Most packages are using ES2020 and Shared Tree is ES2022. I think you can just set the year here (https://eslint.org/docs/latest/use/configure/language-options).